wolox-books/src/app/screens/sign-up/sign-up.component.html

```diff
@@ -12,6 +12,6 @@
         <input class="input-data" formControlName="password" type="password" name="password">
         <button class="sign-up-btn" type="submit">Sign Up</button>
       </form>
-      <button class="login-btn" (click)="sendToLogin()">Login</button>
+      <button class="login-btn" [routerLink]="['/login']">Login</button>
```

Comment:
Same as the other. You should use an `a` tag and it can be `<a routerLink="login">`